### DISCUSSION TITLE
How to actually open the importer?

### DISCUSSION THREAD

**Discussion by Miodec** [firefly-iii/firefly-iii#7480] [Firefly III]
I feel like this is an extra dumb question but how do I open the importer UI? I tried looking for any other discussions and I couldn't find anyone else having this issue... I have followed the instructions to install the Importer during Firefly installation. This docs page shows a screenshot of what looks to be a web interface https://docs.firefly-iii.org/data-importer/how-to-use/select-data-source/ but it doesn't mention the URL. I can't find anything in actual Firefly either (i assumed it would be above the export option but no)

Am i dumb?

**Comment by Miodec** [comment]
The url is localhost:81

**Comment by JC5** [comment]
I've added this to the docs!
